exercises/simple-linked-list/src/example/java/SimpleLinkedList.java

```diff
@@ -0,0 +1,79 @@
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+/**
+ *
+ * @author matt
+ */
+public class SimpleLinkedList {
+
+    class Element {
+
+        int value;
+        Element next;
+
+        Element(int value) {
+            this.value = value;
+        }
+    }
+
+    private Element head;
+    private int size = 0;
```

Comment:
As `0` is already the [default value for an int](https://docs.oracle.com/javase/specs/jls/se7/html/jls-4.html#jls-4.12.5), this initializer is unnecessary.